src/kili/use_cases/utils.py

```diff
@@ -0,0 +1,89 @@
+"""Utils for use cases."""
+from itertools import chain
+from typing import Dict, List, Optional, cast
+
+from kili.adapters.kili_api_gateway import KiliAPIGateway
+from kili.adapters.kili_api_gateway.helpers.queries import QueryOptions
+from kili.core.utils import pagination
+from kili.domain.asset import AssetExternalId, AssetFilters, AssetId
+from kili.domain.asset.helpers import check_asset_identifier_arguments
+from kili.domain.project import ProjectId
+from kili.domain.types import ListOrTuple
+from kili.exceptions import NotFound
+
+
+class UseCasesUtils:
+    """Utils for use cases."""
+
+    def __init__(self, kili_api_gateway: KiliAPIGateway) -> None:
+        """Init."""
+        self.kili_api_gateway = kili_api_gateway
+
+    def get_asset_ids_or_throw_error(
```

Comment:
this method (and maybe others?) are duplicated in the codebase:

- src/kili/use_cases/utils.py
- src/kili/use_cases/asset/utils.py